### PR TITLE
Last flight scenario

### PR DESCRIPTION
**Pull request by Zlorthishen**
#### Summary
Summary: (Content) "Implement Last Flight Scenario, dda pr #43338

#### Purpose of change
Adds a very fun mission that for some reason hasn't been ported yet.

#### Describe the solution
To implement dda pr [#43338](https://github.com/CleverRaven/Cataclysm-DDA/pull/43338) **Co-authored-by: John-Candlebury <John-Candlebury@users.noreply.github.com>**

#### Testing
Created a new character and spawned with no issues.

cataclysm-tiles_2023-02-25_16-42-11

### CODE REVIEW

**Comment by Fris0uman** [comment]
Add this to commit to give proper credit to original author, using co-author is way better to track attribution
```


Co-authored-by: John-Candlebury <John-Candlebury@users.noreply.github.com>
```

File: data/json/scenarios.json [hunk 1]
```diff
@@ -484,6 +484,19 @@
     "professions": [ "miner", "labtech", "security", "bionic_worker", "archaeologist", "drone_op" ],
     "flags": [ "CITY_START", "LONE_START" ]
   },
+  {
+    "type": "scenario",
+    "id": "last_flight",
+    "name": "Last Flight",
+    "points": 2,
+    "description": "Once it became clear that the world was ending you took to the skies, hoping that your flying skills could lead you to safety.  You found no standing safe zones, and when your fuel ran out, were forced to land over an empty bridge.",
```
Comment (Coolthulhu) [inline data/json/scenarios.json:492]: The description says "empty bridge", but the location is a field. Is there a follow up PR that changes one of those?

Comment (Zlorthishen) [reply data/json/scenarios.json:492]: I moved the location to a field because it wasn't spawning properly on the bridge. I'll update the description.

**Comment by Firestorm01X2** [comment]
Scenario works fine. Helicopter has no fuel. Quest mark showed private airport.

**Pull request merged**